schedule.md

```diff
@@ -0,0 +1,89 @@
+# Session plan
+
+## Session 1: R and statistical concepts used
+
+Some of this could be pre-course reading material, see e.g. https://sbfnk.github.io/mfiidd/slides/rintro.pdf
+
+Monday 9:30-11:00
+
+- Introduction to R concepts used in the course (15 mins)
+- Introduction to statistical concepts used in the course and what they look like in stan (30 mins)
+- Practice session: probability distribution and estimates (30 mins)
+- Wrapup (15 mins)
+
+## Session 2: Delay distributions
```

Comment:
Personally, I would favour splitting this as proposed in the learning objects (i.e delays very simply and then biases (i.e censoring and truncation).

Reply:
That would also help as it would make it clear they are general biases one can see and so maybe help people in other areas of their modelling lives

Reply:
> Personally, I would favour splitting this as proposed in the learning objects (i.e delays very simply and then biases (i.e censoring and truncation).

Definitely, that's the idea - just not being clear in the text perhaps.